### PR TITLE
Expand Height on Search Focus

### PR DESCRIPTION
**Pull request by arunim2405**
A quick fix that works and fixes https://github.com/TheWidlarzGroup/rn-emoji-keyboard/issues/43
Can be updated to follow project standards before merging :)

### CODE REVIEW

**Comment by arunim2405** [comment]
Uploading Simulator Screen Recording - iPhone 13 - 2021-11-06 at 20.01.53.mp4…

**Comment by arunim2405** [comment]
https://user-images.githubusercontent.com/47343112/140613324-eec36595-15c4-4a91-ac26-60a630950541.mp4

**Comment by arunim2405** [comment]
Looks smoother than this, idk why it lagged in recording
https://user-images.githubusercontent.com/47343112/140613361-fb40eb74-4ac1-455d-a1eb-68c141770958.mp4

**Comment by jakex7** [comment]
Hi!
Thank you for your contribution, however please follow [CONTRIBUTING.md](https://github.com/TheWidlarzGroup/rn-emoji-keyboard/blob/master/CONTRIBUTING.md) and update your code according to this document (prettier and eslint rules)

**Comment by jakex7** [comment]
Hi @arunim2405 
Is this PR is still in progress? From what i can see it's not compiling and have a lot of errors.
Please let us know if you still want to progress it.

**Comment by arunim2405** [comment]
@jakex7 can I please know what are the errors you're getting? Also it would be super helpful if you can leave review comments on the code which needs to be reformatted.
I just modified the src  of the package in the node_modules of my project and copied similar changes in this pr. Any help would be appreciated.

File: src/components/EmojiStaticKeyboard.tsx [hunk 1]
```diff
@@ -8,7 +8,7 @@ import { Categories } from './Categories'
 import { SearchBar } from './SearchBar'
 import { useKeyboardStore } from '../store/useKeyboardStore'
 
-export const EmojiStaticKeyboard = () => {
+export const EmojiStaticKeyboard = (props) => {
```
Comment (jakex7) [inline src/components/EmojiStaticKeyboard.tsx:11]: Missing types

File: src/components/SearchBar.tsx [hunk 2]
```diff
@@ -1,9 +1,10 @@
 import * as React from 'react'
-import { View, StyleSheet, TextInput, TouchableOpacity } from 'react-native'
+import { View, StyleSheet, TextInput, TouchableOpacity,useWindowDimensions, Animated } from 'react-native'
 import { KeyboardContext } from '../contexts/KeyboardContext'
 import { Icon } from './Icon'
+import { getHeight } from '../utils'
 
-export const SearchBar = () => {
+export const SearchBar = (props) => {
```
Comment (jakex7) [inline src/components/SearchBar.tsx:7]: Missing types

File: src/EmojiPicker.tsx [hunk 3]
```diff
@@ -56,7 +56,7 @@ export const EmojiPicker = ({
                 height: Animated.subtract(height, offsetY),
               },
             ]}>
-            <EmojiStaticKeyboard />
+            <EmojiStaticKeyboard height={height} offsetY={offsetY} onClose={onClose} />
```
Comment (jakex7) [inline src/EmojiPicker.tsx:59]: Using non-existing variables

**Review by jakex7 — COMMENTED** [review]
I think most of the issues appeared after copying files 😉

**Comment by jakex7** [comment]
Mainly issues was:
- No TypeScript and no types
- Using non-existing variables (I think that's the mistake while copying files like you said)
- Eslint and prettier warnings
Every of the above is listed in our [community guidelines](https://github.com/TheWidlarzGroup/rn-emoji-keyboard/blob/master/CONTRIBUTING.md) that i highly encourage to read.
However, thanks to your approach, we have manage to resolve this issue as soon as possible in #47

**Pull request closed without merging**